config/config.go

```diff
@@ -276,6 +289,14 @@ func routeFromSchema(r RouteSchema, index int) (*Route, error) {
 		errors.Add(fmt.Errorf("unknown protocol: %s", r.Protocol))
 	}
 
+	if r.Options != nil {
+		options := r.Options
+		if options.LoadBalancingAlgorithm != "" &&
+			options.LoadBalancingAlgorithm != LeastConns && options.LoadBalancingAlgorithm != RoundRobin {
```

Comment:
it would be more elegant to do a range check here, though not a blocker for me

see https://github.com/cloudfoundry/gorouter/blob/main/config/config.go#L722